docs/xdebug.md

```diff
@@ -0,0 +1,85 @@
+# Debugging PHP in Archipelago
+
+This document describes how to enable Xdebug for local PHP development using the PHPStorm IDE and a docker container running the Archipelago `esmero-php:development` image. It involves interacting with the [esmero/archipelago-docker-images](https://github.com/esmero/archipelago-docker-images) repo and the [esmero/archipelago-deployment](https://github.com/esmero/archipelago-deployment) repo.
+
+## Part 1: Docker
+1. Build the development image with Xdebug enabled. Run the following from the `/archipelago-docker-images/esmero-php-fmp` folder:
```

Comment:
@marlo-longley wondering if we could leave the building part as an extra/advanced item and simply deliver built container (build by us) from docker? What do you think? I feel it is super good to have the "how" a container is build, i really like it, but also, i would love to lower the barrier a bit. Lets talk about this on Monday! Happy new year and thanks for all this good work =)